src/main/java/com/faforever/client/fa/relay/ice/IceAdapterImpl.java

```diff
@@ -262,9 +262,13 @@ private void updateLobbyModeFromGameInfo(GameLaunchMessage gameLaunchMessage) {
   }
 
   @Override
-  @PreDestroy
-  public void stop() {
+  public void destroy() {
     Optional.ofNullable(iceAdapterProxy).ifPresent(IceAdapterApi::quit);
     peer = null;
   }
+
+  public void stop(){
+    //requested by interface
+  }
```

Comment:
This is used by the GameService, converting it to an empty implementation will break that.
You can make `detroy()` call this method.